packages/msw/mixin.core.js

```diff
@@ -0,0 +1,93 @@
+const { Mixin } = require('hops-mixin');
+
+const getMockServiceWorkerContent = () => {
+  const { mkdtempSync, readFileSync } = require('fs');
+  const { tmpdir } = require('os');
+  const { join } = require('path');
+  const { sync: execa } = require('execa');
+
+  const dir = mkdtempSync(join(tmpdir(), 'msw-'));
+
+  execa(process.execPath, [
+    require.resolve('msw/cli'),
+    'init',
+    dir,
+    '--no-save',
+  ]);
+
+  const content = readFileSync(join(dir, 'mockServiceWorker.js'));
+
+  return content;
+};
+
+module.exports = class MswMixin extends Mixin {
+  configureServer(_, middlewares) {
+    if (process.env.ENABLE_MOCKS === 'true') {
```

Comment:
It's minor but in my opinion very helpful to reduce the indentations and keep the code simpler to read.
```suggestion
    if (process.env.ENABLE_MOCKS !== 'true') return;
```